src/website/src/app/documentation/demos/datepicker/demos/datepicker-api.demo.ts

```diff
@@ -1,33 +1,37 @@
 /*
- * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import { Component } from '@angular/core';
 
 const HTML_EXAMPLE = `
-<form clrForm clrLayout="vertical">
-  <clr-date-container>
-    <input type="date" clrDate name="demo" [(ngModel)]="demo">
-  </clr-date-container>
+<form clrForm>
+    <clr-date-container>
+        <label for="basicDemo">Basic Demo</label>
+        <input id="basicDemo" type="date" clrDate name="demo" [(ngModel)]="demo">
+    </clr-date-container>
 </form>
 `;
 
 const MIN_EXAMPLE = `
 <form clrForm clrLayout="vertical">
   <clr-date-container>
+    <label for="minDemo">Min Demo</label>
     <input type="date" clrDate name="demo" [(ngModel)]="demo" min="2019-11-11">
```

Comment:
Missing `id="minDemo"`

Reply:
This probably relates to Jeremy's comments that all these "for"s should better go away.